recipes/react-ssr/server/index.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path');
+
+const Koa = require('koa');
+const statik = require('koa-static');
+
+const DIST_DIR = path.resolve(__dirname, '..', 'dist');
+const renderer = require(path.resolve(DIST_DIR, 'server.js'));
+
+const app = new Koa();
+app.use(renderer);
+app.use(statik(DIST_DIR));
+
+app.listen(3000, () => {
```

Comment:
won't it collide with the "client" server? Both running at 3000.

Reply:
This is production server

Reply:
oh, nvm.